src/main/java/cn/nukkit/item/Item.java

```diff
@@ -995,10 +998,22 @@ private static void initCreativeItems() {
         addCreativeItem(Item.get(Item.FENCE_GATE_ACACIA, 0));
         addCreativeItem(Item.get(Item.FENCE_GATE_DARK_OAK, 0));
         addCreativeItem(Item.get(Item.IRON_BARS, 0));
-        int[] dyeColors = {0, 8, 7, 15, 12, 14, 1, 4, 5, 13, 9, 3, 11, 10, 2, 6};
-        for (int color : dyeColors) {
-            addCreativeItem(Item.get(Item.BED, color));
-        }
+        addCreativeItem(Item.get(Item.BED, DyeColor.WHITE.getWoolData()));
```

Comment:
@PikyCZ This method it's depredicated :(

Reply:
@PikyCZ Use one arraylist for metadata and simple add "addCreativeItem(Item.get(Item.BED, metadata);